Firmware/ultralcd.cpp

```diff
@@ -5428,7 +5428,7 @@ static void lcd_main_menu()
 #endif //MMU_HAS_CUTTER
         } else {
 #ifdef FILAMENT_SENSOR
-            if (fsensor.getAutoLoadEnabled()) {
+            if (fsensor.getAutoLoadEnabled() && fsensor.isEnabled()) {
```

Comment:
Just a nitpick, I would personally put the two things that are && together the other way around, so that if the fsensor is not enabled, then getAutoLoadEnabled() doesn't get called